divan/chat/views.py

```diff
@@ -0,0 +1,291 @@
+from rest_framework.views import APIView
+from django.http import HttpRequest, JsonResponse
+from rest_framework import status
+from rest_framework.response import Response
+from . import serializers
+from . import auth_firebase
+from . import token_process
+from .models import Conversation, Friend, Message, User
+
+
+class Login(APIView):
+    def post(self, request: HttpRequest):
+        email = request.data.get('email')
+        password = request.data.get('password')
+        try:
+            token = auth_firebase.login(email, password)
```

Comment:
as this contains the tokenid and the refresh token
```suggestion
            credentials = auth_firebase.login(email, password)
```